symfony/client/src/orangehrmPerformancePlugin/pages/KpiSave.vue

```diff
@@ -0,0 +1,215 @@
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-divider />
+        <br />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button
+            type="button"
+            display-type="ghost"
+            label="Cancel"
+            @click="onCancel"
+          />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import SwitchInput from '@ohrm/oxd/core/components/Input/SwitchInput';
+import {navigate} from '@ohrm/core/util/helper/navigation';
+import {
+  required,
+  digitsOnly,
+  shouldNotExceedCharLength,
+  minValueShouldBeLowerThanMaxValue,
+} from '@ohrm/core/util/validation/rules';
+import {APIService} from '@/core/util/services/api.service';
+import JobtitleDropdown from '@/orangehrmPerformancePlugin/components/JobtitleDropdown.vue';
+
+const initialKpi = {
+  title: '',
+  jobTitle: null,
+  minRate: null,
+  maxRate: null,
+  isDefault: false,
+};
+
+const rateError = v => {
+  if (digitsOnly(v)) {
+    return (
+      (v >= 0 && v <= 100 && Math.round(v) === v) ||
+      'Should be a number between 0-100'
+    );
+  }
+  return 'Should be a number between 0-100';
+};
+
+export default {
+  name: 'KpiSave',
+  components: {
+    'oxd-switch-input': SwitchInput,
+    'jobtitle-dropdown': JobtitleDropdown,
+  },
+  setup() {
+    const http = new APIService(
+      'https://02594277-e771-4db3-a5ec-ad645ec49c02.mock.pstmn.io',
+      'api/v2/performance/kpi',
+    );
+
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      kpi: {...initialKpi},
+      jobTitle: [
+        {id: 1, label: 'Job 1'},
+        {id: 2, label: 'Job 2'},
+      ],
+      rules: {
+        title: [required, shouldNotExceedCharLength(100)],
+        jobTitle: [required],
+        minRate: [
+          required,
+          rateError,
+          v =>
+            !this.kpi.maxRate ||
+            v <= this.kpi.maxRate ||
+            'Minimum Rating should be less than Maximum Rating',
+        ],
+        maxRate: [
+          required,
+          rateError,
+          minValueShouldBeLowerThanMaxValue(
+            () => this.kpi.minRate,
+            'Maximum Rating should be greater than Minimum Rating',
+          ),
+        ],
+      },
+    };
+  },
+  methods: {
+    onCancel() {
+      navigate('/performance/searchKpi');
+    },
+    onSave() {
+      this.isLoading = true;
+      const jobTitleObj = {
+        id: this.kpi.jobTitle.id,
+        title: this.kpi.jobTitle.label,
+        isDeleted: false,
+      };
+      this.http
+        .create({
+          ...this.kpi,
+          jobTitle: jobTitleObj,
+        })
+        .then(() => {
+          return this.$toast.saveSuccess();
+        })
+        .then(() => {
+          this.onCancel();
+        });
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '@ohrm/oxd/styles/_mixins.scss';
```

Comment:
since both `KpiEdit.vue` and `KpiSave.vue` use same styles, let's add a external style sheet. [refer here](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmPimPlugin/pages/employee/Employee.vue#L380)

Reply:
Made  the external stylesheet based on the reference